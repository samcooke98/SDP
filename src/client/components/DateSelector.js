// import React from "react";


// export default (props) => {
//     return (
//         <span>
//             <label>
//                 {props.label}
//             </label>
//             <input
//                 type="date"
//                 style={
//                     {

//                     }
//                 }
//             />
//         </span>
//     )
// }