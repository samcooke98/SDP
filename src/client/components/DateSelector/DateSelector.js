import React from "react";
import styles from "./web.css";

export default class DateSelector extends React.Component {
    render() {
        return (
            <table className={styles.dateSelectorTable}>
                <tr>
                    <td>
                        FROM
                    </td>
                    <td>
                        <div className={styles.picker}>SELECT</div>
                    </td>
                </tr>
                <tr>
                    <td>
                        TO
                    </td>
                    <td>
                        <div className={styles.picker}>SELECT</div>
                    </td>
                </tr>
            </table>
        )

    }
}