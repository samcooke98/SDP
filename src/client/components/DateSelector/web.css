.container { 
	display: flex;

}

.container label { 
	/* flex-grow: 1; */
	width: 75px;
}

.container input { 
	flex-grow: 3;	
}

.container input { 
	width: 100px;
	height: 20px;
	font-size: 16px;
	margin-bottom: 8px;
	margin-left: 4px;

	font-family: 'Roboto';
	padding: 4px;
}