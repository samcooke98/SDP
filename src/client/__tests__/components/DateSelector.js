import React from "react";
import renderer from "react-test-renderer";
// import ReactDOM from 'react-dom';

import DateSelector from "../../components/DateSelector/DateSelector";


describe('React Component: DateSelector', () => {
    it("matches a snapshot", () => {
        const tree = renderer.create(
            <DateSelector />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})