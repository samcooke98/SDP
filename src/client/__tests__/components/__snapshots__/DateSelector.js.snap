// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Component: DateSelector matches a snapshot 1`] = `
<table
  className={undefined}
>
  <tr>
    <td>
      FROM
    </td>
    <td>
      <div
        className={undefined}
      >
        28/08/17
      </div>
    </td>
  </tr>
  <tr>
    <td>
      TO
    </td>
    <td>
      <div
        className={undefined}
      >
        28/08/17
      </div>
    </td>
  </tr>
</table>
`;
